docs/guides/modules/security/pages/audit-logs.adoc

```diff
@@ -1,22 +1,238 @@
 = Audit logs
-:page-platform: Cloud, Server v4+
-:page-description: "This document describes the ways CircleCI customers can access their audit logs."
+:page-platform: Cloud
```

Comment:
General things - sentence case for all headings and all things you are highlighting in bullet lists